src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -164,6 +166,13 @@ class TestSetup:
     __test__ = False
 
 
+_TEST_PATTERN = re.compile(b"def\\s+test_")
+
+
+def _count_pytest_tests(contents: DigestContents) -> int:
+    return sum([len(_TEST_PATTERN.findall(file.content)) for file in contents])
```

Comment:
Nit: Don't need the list, can sum a generator.


```suggestion
    return sum(len(_TEST_PATTERN.findall(file.content)) for file in contents)
```